grpc/codegen/protobuf_transform.go

```diff
@@ -465,13 +465,24 @@ func convertType(source, target *expr.AttributeExpr, sourceVar string, ta *trans
 		return fmt.Sprintf("%s(%s)", transformHelperName(source, target, ta), sourceVar)
 	}
 
+	sourceType, _ := codegen.GetMetaType(source)
```

Comment:
Would it make sense to move the initialization of `sourceType` under the `if` below?